user-console/src/main/java/org/pentaho/mantle/client/solutionbrowser/SolutionBrowserPanel.java

```diff
@@ -823,4 +826,13 @@ private static final native String encodeUri( String URI )
     return encodeURIComponent(URI);
   }-*/;
 
+  private class InfoDialog extends PromptDialogBox { // All of this is to get a OK button that is aligned right
+
+    public InfoDialog( String title, String message, boolean isHTML, boolean autoHide, boolean modal ) {
+      super( title, Messages.getString( "dialog.button.ok" ), "removeMe", autoHide, modal, isHTML ? new HTML( message ) : new Label( message ) ); //$NON-NLS-1$
+      // This relies on the fact that PromptDialogBox will right justify the buttons if and only if there is a cancel and ok button definition.
+      // We give it a dummy cancel button ("removeMe") and a real OK button to satisfy the contract, then delete the cancel from the panel (parent).
+      cancelButton.removeFromParent();
```

Comment:
The ticket says that the DialogBox text was not left-aligned, does the code fix this issue?

Reply:
This is a one off, does not pertain to other dialogs